cypress/integration/bucket_level_monitor_spec.js

```diff
@@ -302,6 +307,40 @@ describe('Bucket-Level Monitors', () => {
         // Confirm we can see only one row in the trigger list by checking <caption> element
         cy.contains('This table contains 1 row');
       });
+
+      it('to have multiple indices', () => {
+        // Confirm we can see the created monitor in the list
+        cy.contains(SAMPLE_VISUAL_EDITOR_MONITOR);
```

Comment:
We need to modify `SAMPLE_VISUAL_EDITOR_MONITOR` for it to work with `TESTING_INDEX_A` and `TESTING_INDEX_B`. I recommend duplicating it and making it work for the test indices

Reply:
Sorry, in what ways were you thinking it should be modified? Before each of the tests, we delete all of the existing monitors [here](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/259/files/3830f5d08a91f0f8cc5d33bba33eb958f12402c0#diff-e8d1385da5d1a4cf1fb95d35360d2a1485d654b328b057418f4a0c005094911aR273), and then create the sample monitor for editing [here](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/259/files/3830f5d08a91f0f8cc5d33bba33eb958f12402c0#diff-e8d1385da5d1a4cf1fb95d35360d2a1485d654b328b057418f4a0c005094911aR287).

Reply:
Ah yeah, cypress seems to be a little less reliable when it runs as part of the github actions. The tests pass locally when using `yarn run cypress run` and `yarn run cypress open`